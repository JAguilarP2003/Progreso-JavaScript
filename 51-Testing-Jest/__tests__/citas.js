import Citas from '../js/classes/Citas';

describe('Probar la clase de Citas', () => {

    const citas = new Citas();

    test('Agregar una nueva Cita', () => {
        const citaObj = {
            mascota: 'Levi',
            propietario: 'Juan',
            telefono: '123456',
            fecha: '24-08-2021',
            hora:'10:30',
            sintomas: 'No come'
        };

        citaObj.id = Date.now();
        citas.agregarCita(citaObj);

        // Prueba.

        expect(citas).toMatchSnapshot();

    });
})