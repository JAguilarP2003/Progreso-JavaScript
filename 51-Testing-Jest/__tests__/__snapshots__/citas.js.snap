// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de Citas Agregar una nueva Cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "24-08-2021",
      "hora": "10:30",
      "id": 1629851746521,
      "mascota": "Levi",
      "propietario": "Juan",
      "sintomas": "No come",
      "telefono": "123456",
    },
  ],
}
`;
